### PR TITLE
Add new content page for content endpoints and other minor edits.

### PR DESCRIPTION
**Pull request by cmknox**

### CODE REVIEW

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+_Applies to:_
+
+- Windows 11
+- Windows 10
+
+> [!NOTE]
+> All ports are outbound.
+
+To ensure connect delivered via Delivery Optimization and Microsoft Connected Cache is properly configured, the following list of endpoints need to be allowed through the firewall. Use the table below to reference any particular content types supported by Delivery Optimization and Microsoft Connected Cache.
```
Comment (amymzhou) [inline windows/deployment/do/delivery-optimization-endpoints.md:28]: To ensure "content"?

Comment (cmknox) [reply windows/deployment/do/delivery-optimization-endpoints.md:28]: Doh! Good catch, thanks.

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+title: # Delivery Optimization and Microsoft Connected Cache content endpoints.
+description: # The complete list of all fully qualified domain names, ports, and associated content types to use Delivery Optimization and Microsoft Connected Cache.
+ms.date: 07/26/2022
+ms.prod: windows
+ms.technology: windows
+ms.topic: conceptual #reference for complete list of content types, endpoint names, ports, etc.
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:7]:
```suggestion
ms.topic: conceptual #reference for complete list of content types, endpoint names, ports, and more.
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ms.localizationpriority: medium 
+author: # GitHub username (cmknox)
+ms.author: # MS alias (carmenf)
+ms.reviewer: # MS alias of feature PM, optional
+manager: # MS alias of manager (naengler)
+ms.collection: # optional
+- # highpri - high priority, strategic, important, current, etc. articles
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:14]:
```suggestion
- # highpri - high priority, strategic, important, current, and more articles
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+_Applies to:_
+
+- Windows 11
+- Windows 10
+
+> [!NOTE]
+> All ports are outbound.
+
+To ensure content delivered via Delivery Optimization and Microsoft Connected Cache is properly configured, the following list of endpoints need to be allowed through the firewall. Use the table below to reference any particular content types supported by Delivery Optimization and Microsoft Connected Cache.
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:28]:
```suggestion
To ensure content delivered via Delivery Optimization and Microsoft Connected Cache is properly configured, the following list of endpoints needs to be allowed through the firewall. Use this table to reference any particular content types supported by Delivery Optimization and Microsoft Connected Cache.
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+|Domain Name  |Protocol/Port(s)  | Content Type | Additional Information | Version |
+|---------|---------|---------------|-------------------|-----------------|
+| *.b1.download.windowsupdate.com, *.dl.delivery.mp.microsoft.com, *.download.windowsupdate.com, *.au.download.windowsupdate.com, *.au.b1.download.windowsupdate.com, *.tlu.dl.delivery.mp.microsoft.com, *.emdl.ws.microsoft.com, *.ctldl.windowsupdate.com   |  HTTP / 80  | Windows Update Windows Defender Windows Drivers | [Complete list](https://docs.microsoft.com/en-us/windows/privacy/manage-windows-2004-endpoints) of endpoints for Windows Update services and payload. | Microsoft Endpoint Configuration Manager Distribution Point |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:32]:
```suggestion
| *.b1.download.windowsupdate.com, *.dl.delivery.mp.microsoft.com, *.download.windowsupdate.com, *.au.download.windowsupdate.com, *.au.b1.download.windowsupdate.com, *.tlu.dl.delivery.mp.microsoft.com, *.emdl.ws.microsoft.com, *.ctldl.windowsupdate.com   |  HTTP/80  | Windows Update Windows Defender Windows Drivers | [Complete list](/windows/privacy/manage-windows-2004-endpoints) of endpoints for Windows Update services and payload. | Microsoft Endpoint Configuration Manager Distribution Point |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+| *.delivery.mp.microsoft.com  |  HTTP / 80  | Edge Browser | [Complete list](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-security-endpoints) of endpoints for Edge Browser. | Microsoft Endpoint Configuration Manager Distribution Point |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:33]:
```suggestion
| *.delivery.mp.microsoft.com  |  HTTP/80  | Edge Browser | [Complete list](/deployedge/microsoft-edge-security-endpoints) of endpoints for Edge Browser. | Microsoft Endpoint Configuration Manager Distribution Point |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+| *.officecdn.microsoft.com.edgesuite.net, *.officecdn.microsoft.com, *.cdn.office.net |  HTTP / 80  | Office CDN updates | [Complete list](https://docs.microsoft.com/en-us/office365/enterprise/office-365-endpoints) of endpoints for Office CDN updates. | Microsoft Endpoint Configuration Manager Distribution Point |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:34]:
```suggestion
| *.officecdn.microsoft.com.edgesuite.net, *.officecdn.microsoft.com, *.cdn.office.net |  HTTP/80  | Office CDN updates | [Complete list](/office365/enterprise/office-365-endpoints) of endpoints for Office CDN updates. | Microsoft Endpoint Configuration Manager Distribution Point |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+| *.manage.microsoft.com, *.swda01.manage.microsoft.com, *.swda02.manage.microsoft.com, *.swdb01.manage.microsoft.com, *.swdb02.manage.microsoft.com, *.swdc01.manage.microsoft.com, *.swdc02.manage.microsoft.com, *.swdd01.manage.microsoft.com, *.swdd02.manage.microsoft.com, *.swda01-mscdn.manage.microsoft.com, *.swda02-mscdn.manage.microsoft.com, *.swdb01-mscdn.manage.microsoft.com, *.swdb02-mscdn.manage.microsoft.com, *.swdc01-mscdn.manage.microsoft.com, *.swdc02-mscdn.manage.microsoft.com, *.swdd01-mscdn.manage.microsoft.com, *.swdd02-mscdn.manage.microsoft.com |  HTTP / 80, HTTPs / 443  | Intune Win32 Apps | [Complete list](https://docs.microsoft.com/en-us/mem/intune/fundamentals/intune-endpoints) of endpoints for Intune Win32 Apps updates. | Microsoft Endpoint Configuration Manager Distribution Point |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:35]:
```suggestion
| *.manage.microsoft.com, *.swda01.manage.microsoft.com, *.swda02.manage.microsoft.com, *.swdb01.manage.microsoft.com, *.swdb02.manage.microsoft.com, *.swdc01.manage.microsoft.com, *.swdc02.manage.microsoft.com, *.swdd01.manage.microsoft.com, *.swdd02.manage.microsoft.com, *.swda01-mscdn.manage.microsoft.com, *.swda02-mscdn.manage.microsoft.com, *.swdb01-mscdn.manage.microsoft.com, *.swdb02-mscdn.manage.microsoft.com, *.swdc01-mscdn.manage.microsoft.com, *.swdc02-mscdn.manage.microsoft.com, *.swdd01-mscdn.manage.microsoft.com, *.swdd02-mscdn.manage.microsoft.com |  HTTP / 80, HTTPS/443  | Intune Win32 Apps | [Complete list](/mem/intune/fundamentals/intune-endpoints) of endpoints for Intune Win32 Apps updates. | Microsoft Endpoint Configuration Manager Distribution Point |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+| *.statics.teams.cdn.office.net |  HTTP / 80, HTTPs / 443  | Teams | | Microsoft Endpoint Configuration Manager Distribution Point |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:36]:
```suggestion
| *.statics.teams.cdn.office.net |  HTTP/80, HTTPS/443  | Teams | | Microsoft Endpoint Configuration Manager Distribution Point |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+| *.assets1.xboxlive.com, *.assets2.xboxlive.com, *.dlassets.xboxlive.com, *.dlassets2.xboxlive.com, *.d1.xboxlive.com, *.d2.xboxlive.com, *.assets.xbox.com, *.xbl-dlassets-origin.xboxlive.com, *.assets-origin.xboxlive.com, *.xvcb1.xboxlive.com, *.xvcb2.xboxlive.com, *.xvcf1.xboxlive.com, *.xvcf2.xboxlive.com |  HTTP / 80 | Xbox | | Microsoft Endpoint Configuration Manager Distribution Point |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:37]:
```suggestion
| *.assets1.xboxlive.com, *.assets2.xboxlive.com, *.dlassets.xboxlive.com, *.dlassets2.xboxlive.com, *.d1.xboxlive.com, *.d2.xboxlive.com, *.assets.xbox.com, *.xbl-dlassets-origin.xboxlive.com, *.assets-origin.xboxlive.com, *.xvcb1.xboxlive.com, *.xvcb2.xboxlive.com, *.xvcf1.xboxlive.com, *.xvcf2.xboxlive.com |  HTTP/80 | Xbox | | Microsoft Endpoint Configuration Manager Distribution Point |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+| *.tlu.dl.adu.microsoft.com, *.nlu.dl.adu.microsoft.com, *.dcsfe.prod.adu.microsoft.com |  HTTP / 80 | Device Update | [Complete list](https://docs.microsoft.com/en-us/azure/iot-hub-device-update/) of endpoints for Device Update updates.  | Microsoft Endpoint Configuration Manager Distribution Point |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:38]:
```suggestion
| *.tlu.dl.adu.microsoft.com, *.nlu.dl.adu.microsoft.com, *.dcsfe.prod.adu.microsoft.com |  HTTP/80 | Device Update | [Complete list](/azure/iot-hub-device-update/) of endpoints for Device Update updates.  | Microsoft Endpoint Configuration Manager Distribution Point |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+| *.do.dsp.mp.microsoft.com |  HTTP / 80, HTTPs / 443 | Microsoft Connected Cache -> Delivery Optimization Services communication | [Complete list](https://docs.microsoft.com/en-us/windows/deployment/update/waas-delivery-optimization-faq#what-hostnames-should-i-allow-through-my-firewall-to-support-delivery-optimization) of endpoints for Delivery Optimization only.  | Microsoft Connected Cache Managed in Azure |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:39]:
```suggestion
| *.do.dsp.mp.microsoft.com |  HTTP/80, HTTPS/443 | Microsoft Connected Cache -> Delivery Optimization Services communication | [Complete list](/windows/deployment/update/waas-delivery-optimization-faq#what-hostnames-should-i-allow-through-my-firewall-to-support-delivery-optimization) of endpoints for Delivery Optimization only.  | Microsoft Connected Cache Managed in Azure |
```

File: windows/deployment/do/delivery-optimization-endpoints.md [hunk 13]
```diff
@@ -0,0 +1,40 @@
...
+| *.azure-devices.net, *.global.azure-devices-provisioning.net, *.azurecr.io, *.blob.core.windows.net, *.mcr.microsoft.com |  AMQP / 5671, MQTT / 8883, HTTPs / 443 | IoT Edge / IoT Hub communication| [Complete list](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-protocols) of Azure IoT Hub communication protocols and ports. [Azure IoT Guide](https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-devguide-endpoints) to understanding Azure IoT Hub endpoints. | Microsoft Connected Cache Managed in Azure |
```
Comment (JohanFreelancer9) [inline windows/deployment/do/delivery-optimization-endpoints.md:40]:
```suggestion
| *.azure-devices.net, *.global.azure-devices-provisioning.net, *.azurecr.io, *.blob.core.windows.net, *.mcr.microsoft.com |  AMQP/5671, MQTT/8883, HTTPS/443 | IoT Edge/IoT Hub communication| [Complete list](/azure/iot-hub/iot-hub-devguide-protocols) of Azure IoT Hub communication protocols and ports. [Azure IoT Guide](/azure/iot-hub/iot-hub-devguide-endpoints) to understand Azure IoT Hub endpoints. | Microsoft Connected Cache Managed in Azure |
```

File: windows/deployment/do/index.yml [hunk 14]
```diff
@@ -97,4 +97,6 @@ landingContent:
             url: delivery-optimization-workflow.md
           - text: Using a proxy with Delivery Optimization
             url: delivery-optimization-proxy.md
+          - text: Content endpoints for Delivery Optimization / Microsoft Connected Cache
```
Comment (JohanFreelancer9) [inline windows/deployment/do/index.yml:100]:
```suggestion
          - text: Content endpoints for Delivery Optimization/Microsoft Connected Cache
```

File: windows/deployment/do/waas-delivery-optimization-setup.md [hunk 15]
```diff
@@ -31,6 +31,12 @@ Starting with Microsoft Intune version 1902, you can set many Delivery Optimizat
 
 **Starting with Windows 10, version 1903**, you can use the Azure Active Directory (Azure AD) Tenant ID as a means to define groups. To do this set the value for DOGroupIdSource to its new maximum value of 5.
 
+## Allow content endpoints
+
+When using a firewall, it is important that the content endpoints are allowed and associated ports are open. 
+
+[Learn more](delivery-optimization-endpoints.md) about the complete list for fully qualified domains, ports for all Delivery Optimization and Microsoft Connected Cache content types.
```
Comment (JohanFreelancer9) [inline windows/deployment/do/waas-delivery-optimization-setup.md:38]:
```suggestion
[Learn more](delivery-optimization-endpoints.md) about the complete list of fully qualified domains, ports for all Delivery Optimization, and Microsoft Connected Cache content types.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cmknox, kindly commit the suggested changes if you agree with them. Many thanks.

**Pull request merged**